readme.md

```diff
@@ -44,7 +44,7 @@ git commit -m "Adding VividCortex Agents"
 git push heroku master
 ```
 
-You should see that the build was successful. However agents will not start automatically. You will need to start them as a worker process by using the Heroku command. NOTE: Do not scale agents to more than 1. This will put uncessary load on your database and you will be charged for another agent licence.
+You should see that the build was successful. However agents will not start automatically. You will need to start them as a worker process by using the Heroku command. NOTE: Do not scale agents to more than 1. This will put unnecessary load on your database and you will be charged for additional agent licences.
```

Comment:
What about "...and you will be using yet another license that counts towards your quota" or something like that. As written, that sentence sounds like we'll charge people's credit cards automatically, when that's not the case. We'll just use another license from their quota, or reject registration completely. It does cost money, because they can't use the license for something more useful, but it's not an automatic charge, which is the idea that this text conveys to me.